requirements.txt

```diff
@@ -9,7 +9,7 @@ distro
 thoth-analyzer
 yaspin
 texttable
-pyyaml==3.13
+pyyaml==5.3.1
```

Comment:
I'm not sure if it is required to have exactly that version, what about staying in the safe zone `pyyaml >= 4.2b1` due to the CVE? it should install the latest in any case but we are not bound to that specific version. WDYT? @fridex